autoprotocol/protocol.py

```diff
@@ -5034,7 +5034,7 @@ def image_plate(self, ref, mode, dataref):
         """
         return self._append_and_return(ImagePlate(ref, mode, dataref))
 
-    def provision(self, resource_id, dests, volumes):
+    def provision(self, resource_id, dests, volumes=None, amounts=None):
```

Comment:
Would prefer `def provision(self, resource_id, dests, amounts, volumes=None)` instead. This retains backwards compat support while allowing the user to provide mass as the third argument. I.e. `provision(..., ..., Unit("1:gram"))`

Reply:
done, thanks!